src/mbgl/text/glyph_manager.cpp

```diff
@@ -99,8 +99,10 @@ void GlyphManager::processResponse(const Response& res, const FontStack& fontSta
 
         for (auto& glyph : glyphs) {
             auto id = glyph.id;
-            entry.glyphs.erase(id);
-            entry.glyphs.emplace(id, makeMutable<Glyph>(std::move(glyph)));
+            if (!localGlyphRasterizer->canRasterizeGlyph(fontStack, id)) {
```

Comment:
We have similar check inside `GlyphManager::getGlyphs()`, so that  we request only the ranges that cannot be rasterized locally. How would it be possible to have locally rasterizable glyphs in the response?

Reply:
There are a few cases where the 256 glyph range includes glyphs that are locally rasterizable and remotely fetched. 

In such a range, when a glyph is fetched from the remote request, `processResponse` will currently add it to the `entries` table. 

A subsequent call to `getGlyphs` will correctly pass the `canRasterizeGlyph` check, but will return the glyph that was previously written into `entries` by `processResponse`. 

https://github.com/mapbox/mapbox-gl-native/blob/5d1ed5d94194bcbd8c0a66e43e1f10eee0d5f278/src/mbgl/text/glyph_manager.cpp#L35-L38